docs/sources/panels-visualizations/configure-data-links/index.md

```diff
@@ -253,13 +253,17 @@ If you want to add all of the current dashboard's variables to the URL, then use
 1. Hover over any part of the panel to display the menu icon in the upper-right corner.
 1. Click the menu icon and select **Edit** to open the panel editor.
 1. In the panel edit pane, scroll down to the **Data links** section and expand it.
-1. Click **Add link**.
-1. In the dialog box that opens, enter a **Title**. This is a human-readable label for the link, which will be displayed in the UI.
+1. Click **+ Add link**.
+1. In the dialog box that opens, enter a **Title**. This is a human-readable label for the link displayed in the UI.
```

Comment:
Do we want to mention that title and url are now required? ie. the Save button is disabled until we have values.